CHANGES

```diff
@@ -1498,7 +1498,7 @@ www.mcs.anl.gov/mpi/mpich2/mpich2_1_0_6changes.htm.
   output.  Otherwise, the output message will be base on the error message
   associated with the mpi_errno parameter.
 
-- MPID_Segment_init() now takes an additional boolean parameter that specifies
+- MPIR_Segment_init() now takes an additional boolean parameter that specifies
```

Comment:
And now after looking at it again, I originally thought you were doing the reverse, which is why I left it out. Now it looks like the right thing to do, so I'll look at adding it in the next revision. It'll be it's own commit and not part of the massive refactor commit.